### PR TITLE
Anpl 1698 prepare release mq to prod cpanel

### PR DESCRIPTION
**Pull request by ymao2**
This PR is to prepare the aws data-production(sqs) and aws production (cluster) for releasing the new feature of cpanel:  message queue.

### CODE REVIEW

File: terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+  required_version = "~> 1.5"
+}
+
+provider "aws" {
+  alias = "session"
+}
+
+provider "aws" {
+  region = "eu-west-2"
```
Comment (jacobwoffenden) [inline terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf]: Additional components for Analytical Platform/Control Panel should reside in the same region, `eu-west-1`.

File: terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,15 @@
+account_ids = {
+  analytical-platform-data-production       = "593291632749"
+  analytical-platform-management-production = "042130406152"
+}
+
+tags = {
+  business-unit          = "Platforms"
+  application            = "Data Platform"
+  component              = "Message Queue"
```
Comment (jacobwoffenden) [inline terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tfvars]:
```suggestion
  component              = "Message Broker"
```
to match component name

File: terraform/aws/analytical-platform-production/cluster/iam-roles.tf [hunk 3]
```diff
@@ -17,9 +17,8 @@ module "iam_assumable_role_ebs_csi_driver" {
 ##################################################
 
 module "iam_assumable_role_control_panel_api" {
-  source  = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
-  version = "5.28.0"
-
+  source    = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
+  version   = "5.28.0"
   providers = {
```
Comment (jacobwoffenden) [inline terraform/aws/analytical-platform-production/cluster/iam-roles.tf]:
```suggestion
  source    = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
  version   = "5.28.0"

  providers = {
```
Revert this

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]
Functionally OK, but some names could be improved based on https://technical-documentation.data-platform.service.justice.gov.uk/documentation/platform/infrastructure/terraform.html#naming

Also cross region should be removed

File: terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf (hunk 1 above)
Comment (ymao2) [reply terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf]: @jacobwoffenden , I don't mind to change it to `eu-west-1`,  we used `eu-west-2` for firebreak on dev account,  and also had a short discussion with @julialawrence 
https://mojdt.slack.com/archives/C05J081AD98/p1690374209722419

Comment (jacobwoffenden) [reply terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf]: I'm not in favour of splitting control panel components geographically, what happens if eu-west-2 goes does down? does cross region communication add any latency?

I think extensions to control panel should reside in the same region.

@bagg3rs - thoughts?

Comment (bagg3rs) [reply terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf]: It does make sense to keep it in the same region. Keep it simple unless this was to e.g. to improve redundancy.

Comment (ymao2) [reply terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf]: ok, both, let me change it to `eu-west-1`

**Comment by ymao2** [comment]
@jacobwoffenden ,  any hint for where I can see the detail of the `Static Analysis` error ? 😳 ,  I knew you may have mentioned some tips in the past,  but I cannot remember.. :(

File: terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf (hunk 1 above)
Comment (Gary-H9) [reply terraform/aws/analytical-platform-data-production/control-panel-message-broker/terraform.tf]: Resolving as this is done.

**Review by jacobwoffenden — APPROVED** [review]

**Pull request merged**